nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/test/groovy/org/apache/nifi/web/security/x509/ocsp/OcspCertificateValidatorGroovyTest.groovy

```diff
@@ -65,7 +65,8 @@ public class OcspCertificateValidatorGroovyTest {
 
     @Before
     public void setUp() throws Exception {
-        mockProperties = [getProperty: { String propertyName -> return "value_for_${propertyName}" }] as NiFiProperties
+        mockProperties = [getProperty: { String propertyName -> return "value_for_${propertyName}"
+            ,getPropertyKeys: { -> return ["A", "set"] as Set<String>}] as NiFiProperties
```

Comment:
Actually I'm getting weird errors, doesn't look like map coercion likes the getProperty method, possibly because that method is used all over Groovy meta-classes and such. This could replace the above:

mockProperties = new NiFiProperties() {
            @Override
            String getProperty(String key) {
                return 'value_for_' + key
            }

            @Override
            Set<String> getPropertyKeys() {
                return ["A", "set"]
            }
        }